src/Alert/Alert.js

```diff
@@ -6,7 +6,7 @@ import { ALERT_TYPES } from '../common/constants'
 /**
  * Alert Component for Patternfly React
  */
-const Alert = ({ children, onDismiss, type }) => {
+const Alert = ({ children, onDismiss, type, ...props }) => {
```

Comment:
I forgot that we need to define className explicitly to avoid user overriding the className prop we use on the div and also allow user to provide additional classes
```
const Alert = ({ children, className, onDismiss, type, ...props }) => {
```

also please add className to defaultProps.